Task1_Currencies.js

```diff
@@ -3,30 +3,50 @@ const CURRENCY_API_URL = process.env.CURRENCY_API_URL;
 
 
 async function fetchCurrencies() {
-    const currenciesResponse = await fetch(CURRENCY_API_URL);
-     currencies = await currenciesResponse.json();
-     return currencies;
+    try {
+        const currenciesResponse = await fetch(CURRENCY_API_URL);
+        const currencies = await currenciesResponse.json();
+        return currencies;
```

Comment:
you can reduce these 2 lines by returning the following `return await currenciesResponse.json();`